DESCRIPTION

```diff
@@ -1,8 +1,9 @@
 Package: tidyhydat
 Title: Extract and Tidy Canadian 'Hydrometric' Data
-Version: 0.3.2
+Version: 0.4.9000
```

Comment:
Any reason why you incremented to 0.4? My inclination with relatively small change would be to go to 0.3.3.